### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/owenvoke/blade-fontawesome/pull/33/commits/de01d160e42e5fa827354a2b4e0bdd783e125bcf?w=1) for Laravel 9. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/blade-fontawesome.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "owenvoke/blade-fontawesome": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: .github/workflows/tests.yml
Comment (jasonmccreary) [inline .github/workflows/tests.yml:11]: Looks like it dropped `8.1`. You should probably re-add it. I'll fix that up for the future.

Comment (owenvoke) [reply .github/workflows/tests.yml]: Thanks for this! 🥳

**Review by owenvoke — APPROVED** [review]
Just tweaked a few bits, now LGTM! 🥳 (One thing that wasn't updated was `symfony/var-dumper`, I guess maybe that should be updated to `^6.0` for L9?)

Not sure whether I want to drop support for Laravel 8, but will think about it before merging this. 👍🏻 Cheers!

**Comment by jasonmccreary** [comment]
@owenvoke, thanks for the feedback. We'll get the Symfony dependencies tweaked for next time.

**Pull request merged**
